docs/flow/modin/data_management/factories.rst

```diff
@@ -0,0 +1,19 @@
+Factories Module Description
+""""""""""""""""""""""""""""
+
+Brief description
+'''''''''''''''''
+Modin has many execution backends. Calling any DataFrame API function will end up in some backend-specific method. The responsibility of dispatching high-level API calls to backend-specific function lays on :doc:`QueryCompiler </flow/modin/backends/query_compiler.rst`. Building the `QueryCompiler` from external data is the role of the factories module. The mission of this module is to route IO function calls from the API level to its actual backend-specific implementations, which builds `QueryCompiler` of the corresponding backend.
+
+Backend representation via Factories
+''''''''''''''''''''''''''''''''''''
+Backend is a combination of the `QueryCompiler` and `Execution engine`. For example, ``PandasOnRay`` backend means the combination of the ``PandasQueryCompiler`` and ``Ray`` engine. 
+
+In the scope of this module, each backend is represented with a factory class located in ``factories.py``. Factory contains the IO module of the corresponding backend and is responsible for dispatching calls of IO functions to their actual implementations in an underlying IO module. For more information about IO module visit :doc:`related doc </flow/modin/engines/base/io.rst>`.
```

Comment:
```suggestion
In the scope of this module, each backend is represented with a factory class located in ``factories.py``. Factory contains the IO module of the corresponding backend and is responsible for dispatching calls of IO functions to their actual implementations in an underlying IO module. For more information about IO module visit :doc:`related doc </flow/modin/engines/base/frame/io.rst>`.
```

Reply:
going by #2952 the future path of the IO module docs will be exactly as it's written https://github.com/modin-project/modin/pull/2952/files#diff-4ce064cf82a7c9548037a6cae9cbf0f23ea6a30856feee35182cd0e6bf00743f